src/main/java/com/codessquad/qna/web/domain/Question.java

```diff
@@ -13,14 +10,19 @@
 
     @Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
-    private Long id;
+    private long id;
 
-    private String writer;
+    @ManyToOne
+    @JoinColumn(foreignKey = @ForeignKey(name = "fk_questions_writer"))
+    private User writer;
+    @Column(length = 45)
```

Comment:
요 두 줄 사이에는 한 줄의 줄바꿈이 있으면 좋을 것 같네요.

```suggestion
    private User writer;

    @Column(length = 45)

Reply:
넵 수정했습니다!